src/routes/retro-certs.js

```diff
@@ -35,6 +35,7 @@ function createRouter() {
     if (formRecord) {
       responseJson.formData = formRecord.formData;
       responseJson.confirmationNumber = formRecord.confirmationNumber;
+      responseJson.authToken = formRecord.authToken;
```

Comment:
You don't need the actual authToken value, right? Maybe just return !!formRecord.authToken? This reduces the risk of the token leaking and someone using it to log in as the user.

Reply:
I was just going to say this, it doesn't make any difference in practice since it's staff view not admin etc, but we can return a boolean like hasLoggedIn and then update this line to read hasLoggedIn: https://github.com/cagov/unemployment/blob/de6546488743e61b22aa77c6426bcb30a8768ed9/src/client/pages/StaffViewConfirmationPage/index.js#L35

(I considered doing that but thought authToken was being returned already and didn't check that it was 😓)

Reply:
Sure. Yeah, I thought about this a bit too... does the authToken really help you log in that much? Particularly if you already have the three login fields?

Reply:
I don't think it really matters in practice! better in principle and for readability though (it's clearer that we don't actually use the token itself)